City-of-Helsinki/drupal-infofinland
# Feature/IFU-315-content-migration (Page migration)

**How to test**
```
composer install;
make shell;
drush deploy;
```
Run `drush ms` to see migration status.
Run 
```
drush migrate-import content_import_pages_to_nodes_from_csv_fi;
drush migrate-import content_import_pages_to_nodes_from_csv_en;
drush migrate-import content_import_pages_to_nodes_from_csv_translations;
```
Check that there are no errors and pages are imported. Some translations files might be ignored because of missing title and other data.

Go to: `/fi/admin/config/search/path/update_bulk`
Select `Content` and `Update the URL alias for paths having an old URL alias` option. Click Update button.

Go to: `/fi/admin/content`

Check that the page and translations are working. 
Check that URL alias (can be found from right sidepanel) is generated in english for every translation and is identical between every translation.
